chat/consumers.py

```diff
@@ -0,0 +1,43 @@
+import json
+from channels.generic.websocket import AsyncWebsocketConsumer
+
+class ChatConsumer(AsyncWebsocketConsumer):
+    """
+    WebSocket consumer for handling real-time chat messages in specific rooms.
+    """
```

Comment:
Class name ChatConsumer is descriptive; consider adding a base docstring or type annotation for clarity.